tests/docker/BUILD

```diff
@@ -358,9 +358,13 @@ docker_push_all(
     bundle = ":three_images_bundle",
 )
 
-container_test(
-    name = "basic_dockerfile_image",
-    configs = ["//tests/docker/configs:basic_dockerfile.yaml"],
+[container_test(
```

Comment:
Lets move these tests (if possible) to GCB (sync with Alex that is also doing this for some other tests)

Reply:
Discussed offline. These tests are running on Buildkite + RBE and runs pretty fast.